app/models/concerns/control_objective_items/finding_pdf_data.rb

```diff
@@ -39,51 +50,52 @@ def get_initial_finding_attributes finding, hide, show
       end
 
       if finding.origination_date.present? && hide.exclude?('origination_date')
-        body << "<b>#{finding.class.human_attribute_name('origination_date')}:"+
-          "</b> #{finding_origination_date_text_for finding}\n"
+        body << "<b>#{custom_labels[:origination_date] || finding.class.human_attribute_name('origination_date')}: " +
```

Comment:
El espacio que has puesto ya está después del `</b>`.
```suggestion
        body << "<b>#{custom_labels[:origination_date] || finding.class.human_attribute_name('origination_date')}:" +
```